### ISSUE TITLE
Undefined array key "completion_tokens"

### ISSUE THREAD

**Issue by ManukMinasyan** [openai-php/client#18]
Hi guys. Thanks for your work. I am getting this kind of error.

[https://flareapp.io/share/Lm8NrdV7#F2](https://flareapp.io/share/Lm8NrdV7#F2)

**Comment by gehrisandro** [comment]
Hi @ManukMinasyan 

I was not able to reproduce the issue.
Additionally the API specification says, that "completion_tokens" is not nullable. See: https://github.com/openai/openai-openapi/blob/master/openapi.yaml#L2080

Can you provide more details about the parameters you sent?

**Comment by ManukMinasyan** [comment]
After many time of searches, I have found the problem. It turns out that OpenAI will not return completion_tokens.

Request:
```curl
curl --location --request POST 'https://api.openai.com/v1/completions' \
--header 'Authorization: Bearer XXXXXXXXXXXXXX' \
--header 'Content-Type: application/json' \
--data-raw '{
    "model": "text-davinci-002",
    "prompt": "Instruction: <<Create an article idea title and outline for an essay about the following keyword>> Keyword:<<sasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdsasd,.asmdv>>Structure: <<Title:Outline:>>",
    "max_tokens": 100,
    "temperature": 0.7
}'
```

Response

```json
{
    "id": "cmpl-6BQocmU7pbkVMGcRFXaGcgtnLJ259",
    "object": "text_completion",
    "created": 1668182706,
    "model": "text-davinci-002",
    "choices": [
        {
            "text": "",
            "index": 0,
            "logprobs": null,
            "finish_reason": "stop"
        }
    ],
    "usage": {
        "prompt_tokens": 116,
        "total_tokens": 116
    }
}
```

**Comment by ManukMinasyan** [comment]
I think it is necessary to set completion tokens to zero in this situation.

**Comment by gehrisandro** [comment]
@ManukMinasyan Thank you for clarification.

Now I know, why I wasn't able to reproduce the bug: It has already been fixed in Release 0.2.0
Please make sure, that you are using the latest version of the package.
